tools/deployment-cli-tools/tests/resources/applications/migration_app/tasks/sub-task/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG CLOUDHARNESS_BASE_DEBIAN
+FROM $CLOUDHARNESS_BASE_DEBIAN
```

Comment:
this is a not realistic test case: if apk is there, CLOUDHARNESS_BASE was used

Reply:
for @ddelpiano add a migration step:
- keywork will be cloudharness_base
- the replacement will be apt-get in favour of apk, since apk is not used anymore and the new  package manager used by cloudharness base is apt-get (since this moved to debian)